File: bionic-js-tool/testing-code/swift/project-with-host-files/HostProject/host/BjsMotorVehicles/BjsBeautifulVehicles.swift
import Foundation
import Bjs

@objc(BjsBeautifulVehicles)
class BjsBeautifulVehicles : BjsProject {
    
    override class func initialize(_ bjs: Bjs) {
        bjs.loadBundle(BjsBeautifulVehicles.self, "MotorVehicles")
        bjs.addNativeWrapper(BaseEngineBjsWrapper.self)
        bjs.addNativeWrapper(EngineBjsWrapper.self)
    }
}

File: bionic-js-tool/testing-code/java/project-with-host-files/HostProject/src/main/java/test/project/host/BjsMotorVehicles/BjsBeautifulVehicles.java
package test.project.host.BjsMotorVehicles;

import bionic.js.Bjs;
import bionic.js.BjsProject;
import bionic.js.BjsProjectTypeInfo;

public class BjsBeautifulVehicles extends BjsProject {
    
    @BjsProjectTypeInfo.Initializer
    public static void initialize(Bjs bjs) {
        initProject();
        bjs.loadBundle(BjsBeautifulVehicles.class, "MotorVehicles");
        bjs.addNativeWrapper(test.project.host.native.BaseEngineBjsExport.Wrapper.class);
        bjs.addNativeWrapper(test.project.host.native.EngineBjsExport.Wrapper.class);
    }
}